torch/quantization/fuse_modules.py

```diff
@@ -45,12 +48,19 @@ def fuse_conv_bn_relu(conv, bn, relu):
     """
     assert(conv.training == bn.training == relu.training),\
         "Conv and BN both must be in the same mode (train or eval)."
-
     if conv.training:
-        return torch_fused.ConvBnReLU2d(conv, bn, relu)
+        assert not relu.inplace, 'We only support fusion of non-inplace ReLU.'
+        if isinstance(conv, torch.nn.Conv3d):
```

Comment:
You also need to add a qat module here for training support. https://github.com/pytorch/pytorch/blob/master/torch/nn/intrinsic/qat/modules/conv_fused.py#L156

Please make sure that conv3d+ bn fusion is supported for both training and eval.

Reply:
Let's add it in another PR. This PR is mainly for post-training quantization flow.